New page: aria-readonly (#10762)
mdn/content: files/en-us/web/accessibility/aria/attributes/aria-readonly/index.md

---
title: 'aria-readonly'
slug: Web/Accessibility/ARIA/Attributes/aria-readonly
tags: 
  - Accessibility
  - ARIA
  - ARIA attribute
  - ARIA property
  - aria-readonly
  - Reference
---

The `aria-readonly` attribute indicates that the element is not editable, but is otherwise operable. 

## Description

> When you want to indicate that an interactive element works but is not editable, set `aria-readonly="true"`. This indicates to the user that an element that would normally be focusable and copyable still is, but it is read only. Not disabled.

Reviewer: ```suggestion
When you want to indicate that an interactive element works but is not editable, set `aria-readonly="true"`. This indicates to the user that an interactive element that would normally be focusable and copyable has been placed in a read-only (not disabled) state.
```

> When `aria-readonly` is set to true, it means the user can read but not set the value of the widget. Readonly elements, as opposed to disabled elements, are relevant to the user, so you should not prevent the user from navigating to the element or its focusable descendants or copying the value. 

Reviewer: ```suggestion
When `aria-readonly` is set to `true`, it means the user can read but not set the value of the widget. Read-only elements are still relevant to the user, so you should not prevent the user from navigating to the element or its focusable descendants or copying the value. 
```

> Examples include:

Reviewer: ```suggestion
Examples include:

```
